tests/integration_tests/functional/test_balloon.py

```diff
@@ -37,13 +37,11 @@ def get_rss_from_pmap():
     return second_rss
 
 
-def make_guest_dirty_memory(ssh_connection, should_oom=False, amount=8192):
```

Comment:
All the changes of test_balloon.py included in "[test: use new CI artifacts and drop dynamic fixtures](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/3630867b4dec8f53189afa56a32f9ffe90b69feb)" don't match the concept of the commit, because it just makes tests stable.

Reply:
fixed it on my forked repo by combining "[test: use new CI artifacts and drop dynamic fixtures](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/3630867b4dec8f53189afa56a32f9ffe90b69feb)" and "[test: lower threshold for a stable RSS](https://github.com/firecracker-microvm/firecracker/pull/3896/commits/c6e48fe3f012cfec32daffe6bc04948eb5527f44)" into 1.